### PR TITLE
Support chaining Path commands

### PR DESCRIPTION
**Pull request by nmay231**
Allow chaining path operations like the following:

```diff
- p.M(20, -40)
- p.L(20, -27)
- p.L(0, -20)
+ p.M(20, -40).L(20, -27).L(0, -20)
```

I modified `Path.append` to return `self` instead of each method individually so it would keep them all on one line, but maybe that's just kicking the formatting bucket down the road. Let me know what you would prefer :+1:

### CODE REVIEW

File: drawSvg/elements.py [hunk 1]
```diff
@@ -549,30 +549,31 @@ def append(self, commandStr, *args):
...
+    def S(self, cx2, cy2, ex, ey): return self.append('S', cx2, -cy2, ex, -ey)
+    def s(self, cx2, cy2, ex, ey): return self.append('s', cx2, -cy2, ex, -ey)
+    def Q(self, cx, cy, ex, ey): return self.append('Q', cx, -cy, ex, -ey)
+    def q(self, cx, cy, ex, ey): return self.append('q', cx, -cy, ex, -ey)
+    def T(self, ex, ey): return self.append('T', ex, -ey)
+    def t(self, ex, ey): return self.append('t', ex, -ey)
     def A(self, rx, ry, rot, largeArc, sweep, ex, ey):
-        self.append('A', rx, ry, rot, int(bool(largeArc)), int(bool(sweep)), ex,
+        return self.append('A', rx, ry, rot, int(bool(largeArc)), int(bool(sweep)), ex,
                     -ey)
```
Comment (cduck) [inline drawSvg/elements.py:574]: Wrap this to 80.

```suggestion
        return self.append('A', rx, ry, rot, int(bool(largeArc)),
                           int(bool(sweep)), ex, -ey)
```

File: drawSvg/elements.py [hunk 2]
```diff
@@ -549,30 +549,31 @@ def append(self, commandStr, *args):
...
     def a(self, rx, ry, rot, largeArc, sweep, ex, ey):
-        self.append('a', rx, ry, rot, int(bool(largeArc)), int(bool(sweep)), ex,
+        return self.append('a', rx, ry, rot, int(bool(largeArc)), int(bool(sweep)), ex,
                     -ey)
```
Comment (cduck) [inline drawSvg/elements.py:577]:
```suggestion
        return self.append('a', rx, ry, rot, int(bool(largeArc)),
                           int(bool(sweep)), ex, -ey)
```

File: drawSvg/elements.py [hunk 3]
```diff
@@ -586,6 +587,7 @@ def arc(self, cx, cy, r, startDeg, endDeg, cw=False, includeM=True,
         elif includeM:
             self.M(cx+sx, cy+sy)
         self.A(r, r, 0, largeArc ^ cw, cw, cx+ex, cy+ey)
+        return self
```
Comment (cduck) [inline drawSvg/elements.py:590]: This can just be `return self.A(`.

File: README.md [hunk 4]
```diff
@@ -78,9 +78,9 @@ arrow = draw.Marker(-0.1, -0.5, 0.9, 0.5, scale=4, orient='auto')
 arrow.append(draw.Lines(-0.1, -0.5, -0.1, 0.5, 0.9, 0, fill='red', close=True))
 p = draw.Path(stroke='red', stroke_width=2, fill='none',
               marker_end=arrow)  # Add an arrow to the end of a path
-p.M(20, -40)
-p.L(20, -27)
-p.L(0, -20)
+
+# Chain multiple path operations
+p.M(20, -40).L(20, -27).L(0, -20)
```
Comment (cduck) [inline README.md:83]: Remove the blank line and move the comment to the same line as the path commands.  This should make it clear this is part of the draw arrows section.

**Comment by nmay231** [comment]
Made the changes. Let me know if there's anything else :+1:

**Comment by cduck** [comment]
Published version 1.6.1 to PyPI.

**Pull request merged**